docs/sts/assume-role.md

```diff
@@ -10,19 +10,28 @@ The temporary security credentials returned by this API consists of an access ke
 #### DurationSeconds
 The duration, in seconds. The value can range from 900 seconds (15 minutes) up to 12 hours. If value is higher than this setting, then operation fails. By default, the value is set to 3600 seconds.
 
-| Params | Value |
-| :-- | :-- |
-| *Type* | *Integer* |
+| Params        | Value                                           |
+| :--           | :--                                             |
+| *Type*        | *Integer*                                       |
 | *Valid Range* | *Minimum value of 900. Maximum value of 43200.* |
-| *Required* | *No* |
+| *Required*    | *No*                                            |
+
+#### Policy
+An IAM policy in JSON format that you want to use as an inline session policy. This parameter is optional. Passing policies to this operation returns new temporary credentials. The resulting session's permissions are the intersection of the canned policy name and the policy set here. You cannot use this Policy to grant more permissions than those allowed by the canned policy name being assumed.
+
```

Comment:
You cannot use this Policy -> You cannot use this policy

Reply:
Use GitHub suggestions for docs easy to merge into PRs.